test/cli/commands_test.py

```diff
@@ -1256,44 +1256,40 @@ def test_encoding(encoding_in, encoding_out):
         )
 
 
-def test_cli_pass_on_correct_encoding_argument():
+@pytest.mark.parametrize(
+    "encoding,method,expect_success",
+    [
+        ("utf-8", "command-line", False),
+        ("utf-8-SIG", "command-line", True),
+        ("utf-8", "config-file", False),
+        ("utf-8-SIG", "config-file", True),
```

Comment:
For completeness we could add the following, as capitalisation seems to be irrelevant:

```suggestion
        ("utf-8", "command-line", False),
        ("UTF-8", "command-line", False),
        ("utf-8-sig", "command-line", True),
        ("UTF-8-SIG", "command-line", True),
        ("utf-8-SIG", "command-line", True),
        ("utf-8", "config-file", False),
        ("UTF-8", "config-file", False),
        ("utf-8-sig", "config-file", True),
        ("UTF-8-SIG", "config-file", True),
        ("utf-8-SIG", "config-file", True),
```

Reply:
Makes sense, but I think I'll omit it for now, as encoding is something SQLFluff doesn't really process, it mainly relies on the `chardet` package and Python built-ins like file, etc.